app/controllers/sessions_controller.rb

```diff
@@ -2,7 +2,7 @@ class SessionsController < ApplicationController
 
   def create
     domain = env["omniauth.auth"]["info"]["email"].split('@').last
-    allowed_domains = ['raybeam.com']
+    allowed_domains = ENV.fetch('allowed_domains').split(',')
```

Comment:
It is pulled from the YAML file into ENV in [config/application.rb#L9](https://github.com/Raybeam/zartan/blob/master/config/application.rb#L9).  I'm OK with making allowed_domains an array in the YAML file though.